tests/framework/experiment/experiment.py

```diff
@@ -0,0 +1,48 @@
+import os
+from unittest import TestCase
+
+import torch
+from torch import nn
+
+from poutyne import Experiment
+
+test_checkpoints_path = "./tests_checkpoints"
+ckpt_1_path = os.path.join(test_checkpoints_path, "checkpoint_epoch_1.ckpt")
+ckpt_last_path = os.path.join(test_checkpoints_path, "checkpoint.ckpt")
+
+
+class ExperimentTest(TestCase):
+
+    def setUp(self):
+        self.pytorch_network = nn.Linear(1, 1)
+        self.optimizer = torch.optim.SGD(self.pytorch_network.parameters(), lr=1e-3)
+        self.loss_function = nn.MSELoss()
+
+        self.test_experiment = Experiment(test_checkpoints_path,
+                                          self.pytorch_network,
+                                          optimizer=self.optimizer,
+                                          loss_function=self.loss_function,
+                                          monitor_metric="loss",
+                                          monitor_mode="max")
```

Comment:
Yeah, as we discussed, I think we could really just train for 1 epoch  in setUp. This adds a few lines for the creation of the temporary directory and removes the files for the checkpoints in the git repo.